dask/dataframe/core.py

```diff
@@ -4225,23 +4226,23 @@ def is_monotonic_decreasing(self):
 
 class DataFrame(_Frame):
     """
-    Parallel Pandas DataFrame
+        Parallel Pandas DataFrame
+    from typing import Type
+        Do not use this class directly.  Instead use functions like
+        ``dd.read_csv``, ``dd.read_parquet``, or ``dd.from_pandas``.
```

Comment:
It looks like something went wrong with the docstring here, an import statement and an extra level of indentation.

Reply:
Yes, my mistake, probably a wrong copy-paste.

Reply:
Should be fine now.